packages/@sanity/core/src/actions/dataset/listDatasetCopyJobs.js

```diff
@@ -0,0 +1,90 @@
+module.exports = async function listDatasetCopyJobs(args, context) {
```

Comment:
🏳️ (don't read too much into this): 

The only comment I could leave here is that, perhaps we should test this a bit, as there are three different responsibilities:

1. Data fetching: are the parameters right? do communications succeed?
2. Data mapping: is the returned payload presentable? does it have the right information?
3. Data presenting: can we present the information?

Refactoring this into three different units will be overkill, but perhaps we can build some test around how it would behave when requests fail.

Reply:
The only thing we can verify here with a unit test is presenting the information. I did try to write a test for that a while ago but I couldn't find a way to verify what is being printed to the console. 
The rest of the things you mentioned would be in an integration test I guess but that's out of scope for this PR. 🤔